examples_rclcpp/src/action_server/lightbulb.cpp

```diff
@@ -0,0 +1,98 @@
+#include "action_server/lightbulb.hpp"
+
+using namespace robotis;
+
+LightBulb::LightBulb() : Node("light")
```

Comment:
Add new line to set new variable

Reply:
Added new line before set new variable and push